frontend/src/core/codemirror/completion/Autocompleter.ts

```diff
@@ -60,7 +60,6 @@ export const Autocompleter = {
           info: () => constructCompletionInfoNode(option.completion_info),
         };
       }),
-      validFor: /^\w*$/,
```

Comment:
I am concerned about the performance hit from this.

Instead of deleting, can we fix this regex? From looking at the video in your issue, I wonder if ` /^\w+$/` would be better.

Reply:
thanks @akshayka , you are right about the performance hit.
I've been a little stuck because any regex doesn't work well with other libraries, eg polars df
```
df = pl.DataFrame({
    "a": [1,2]
})
df.clo# <---- cursor here doesn't autocomplete
```

I don't know why I'm so stuck unfortunately 🥲